lib/piggly/parser/grammar.tt

```diff
@@ -359,7 +359,7 @@ grammar Piggly
   end
 
   rule dollarQuoteMarker
-    '$' tag:( [a-z\200-\377_] [a-z\200-\377_0-9]* )? '$' <Piggly::Parser::Nodes::TDollarQuoteMarker>
+    '$' tag:( [a-z\w_] [a-z\w_0-9]* )? '$' <Piggly::Parser::Nodes::TDollarQuoteMarker>
```

Comment:
Perhaps this should this be `[[:alpha:]][[:word:]]*`? According to https://ruby-doc.org/core-2.1.1/Regexp.html , `\w` only matches ASCII characters. I'm not sure if PL/pgSQL allows unicode identifiers, but I don't remember why I matched non-ASCII chars here in the first place!